### PR TITLE
Codelabs Core & Unit Tests

### PR DESCRIPTION
**Pull request by cjiang2000**
Test: Manual

### CODE REVIEW

File: src/core/search/searchapi.js [hunk 1]
```diff
@@ -242,4 +242,36 @@ export default class SearchApi {
...
       .then(response => response.json());
   }
+
+  /**
+ * Initiate a search
+ * @param {boolean} includeLastName include last name in the search
+ * @param {boolean} useUserName use username instead of real name
+ * @param {string} additionalSearchText more text to add to the search
+ */
+Search(includeLastName, useUserName, additionalSearchText) {
```
Comment (cea2aj) [inline src/core/search/searchapi.js]: Based on what I recall from the codelab, I think the function name is supposed to have your name in it. Something like 'chrisSearch'. Also for function names, we use camelCase so we don't capitalize the first letter

File: src/core/search/searchapi.js [hunk 2]
```diff
@@ -242,4 +242,36 @@ export default class SearchApi {
     return request.get()
       .then(response => response.json());
   }
+
+  /**
+ * Initiate a search
```
Comment (cea2aj) [inline src/core/search/searchapi.js]: I think your indentation is a little off compared to the other functions

File: src/core/search/searchapi.js (hunk 1 above)
Comment (cjiang2000) [reply src/core/search/searchapi.js]: Wow, good call Connor. I'll be sure to make the changes.

File: src/core/search/searchapi.js (hunk 2 above)
Comment (cjiang2000) [reply src/core/search/searchapi.js]: I will make sure to stick to semistandard regulations!

File: src/core/search/searchapi.js (hunk 1 above)
Comment (cjiang2000) [reply src/core/search/searchapi.js]: The fixed have been implemented. Thoughts?

**Review by oshi97 — COMMENTED** [review]
wow, looks great! +1 from me!!

File: src/core/search/searchapi.js (hunk 1 above)
Comment (cea2aj) [reply src/core/search/searchapi.js]: looks good!

**Review by cea2aj — APPROVED** [review]

**Pull request merged**